dist/parcoords.standalone.js

```diff
@@ -9355,6 +9403,34 @@
       };
     };
 
+    // mark an array of data
+    var mark = function mark(config, pc, canvas, events, ctx, position) {
+      return function () {
+        var data = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : null;
```

Comment:
[ES6's default parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) can simplify this:
```
const highlight = (config, pc, canvas, events, ctx, position) =>
  function(data = null) {
    if (data === null) {
      return config.highlighted;
    }
```